arch/xtensa/Kconfig

```diff
@@ -13,6 +13,7 @@ config ARCH_CHIP_ESP32
 	bool "Espressif ESP32"
 	select ARCH_FAMILY_LX6
 	select XTENSA_HAVE_INTERRUPTS
+	select ARCH_HAVE_FPU
```

Comment:
@Ouss4 has esp32 FPU?

Reply:
Yes, ESP32 and ESP32-S3 both have an FPU coprocessor.  S3 has other coprocessors as well.

Reply:
Ok, @zhuyanlinzyl please add riscv_fpucmp.c to Make.defs too